libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/TriggerConditions/Microsoft.OnTypingActivity.schema

```diff
@@ -1,8 +1,8 @@
 {
     "$schema": "https://raw.githubusercontent.com/microsoft/botbuilder-dotnet/master/schemas/component.schema",
     "$role": "union(Microsoft.ITriggerCondition)",
-    "title": "On Typing activity",
```

Comment:
>title": "On Typing activity", [](start = 5, length = 29)

Little weird from a history standpoint to rename--nothing to do with typing, right?

Reply:
good catch, didn't mean to replace

---
In reply to: [360621601](https://github.com/microsoft/botbuilder-dotnet/pull/3174#discussion_r360621601) [](ancestors = 360621601)